src/main/resources/META-INF/plugin.xml

```diff
@@ -118,7 +118,7 @@
 
     <extensions defaultExtensionNs="com.intellij">
         <codeInsight.linkHandler prefix="#sonarissue/" handlerClass="org.sonarlint.intellij.editor.SonarLinkHandler"/>
-        <toolWindow id="SonarLint" anchor="bottom" icon="/images/ico-sonarlint-13.png" factoryClass="org.sonarlint.intellij.ui.SonarLintToolWindowFactory"/>
+        <toolWindow id="SonarLint" anchor="bottom" icon="/images/ico-sonarlint-13.png" factoryClass="org.sonarlint.intellij.ui.SonarLintToolWindowFactory" canCloseContents="true"/>
```

Comment:
Does it mean all tabs could be closed now? Or only hotspots?

Reply:
No because I also call `setCloseable(false);` on tabs that we don't want to be closeable. In the end only the hotspots tab is closeable